### ISSUE TITLE
ruby 2.4.0 Fixnum is deprecated

### ISSUE THREAD

**Issue by NuclearMachine** [mislav/will_paginate#528]
just switched from ruby 2.3 to ruby-2.4:

```
home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:27: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:91: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:27: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:91: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:27: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:91: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:27: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:91: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:27: warning: constant ::Fixnum is deprecated
/home/nuclearmachine/.rvm/gems/ruby-2.4.0/gems/will_paginate-3.1.0/lib/will_paginate/view_helpers/link_renderer.rb:91: warning: constant ::Fixnum is deprecated

```

**Comment by NuclearMachine** [comment]
It seems that I can fix the problem by manually editing link_renderer.rb...... Any plans of fixing this bug?

**Comment by mrhead** [comment]
This is a duplicate of https://github.com/mislav/will_paginate/issues/523.

**Comment by mislav** [comment]
Fixed in v3.1.6
